StoreTestWPF.ViewModel/ViewModels/CakeViewModel.cs

```diff
@@ -0,0 +1,66 @@
+﻿using StoreTestWPF.DAL.Models;
+
+namespace StoreTestWPF.ViewModel.ViewModels
+{
+    public class CakeViewModel : ViewModelBase
+    {
+        private Cake cake;
+
+        public CakeViewModel(Cake cake)
```

Comment:
Please do not forget to validate input parameters for null.

Reply:
Fixed